contracts/src/token/memecoin.cairo

```diff
@@ -61,12 +81,16 @@ mod UnruggableMemecoin {
     struct Storage {
         marker_v_0: (),
         team_allocation: u256,
+        pre_launch_holders_count: u8,
         tx_hash_tracker: LegacyMap<ContractAddress, felt252>,
         transfer_restriction_delay: u64,
         launch_time: u64,
+        launch_block_number: u64,
+        launch_liquidity_parameters: Option<LiquidityParameters>,
         factory_contract: ContractAddress,
         liquidity_type: Option<LiquidityType>,
         max_percentage_buy_launch: u16,
+        launch_liquidity_base_amount: Option<u256>,
```

Comment:
isn't this total_supply - team_allocation?

Reply:
Removed.